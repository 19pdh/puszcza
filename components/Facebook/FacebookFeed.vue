<template>
  <section class="feed">
    <h1>Zobacz co siÄ™ dzieje!</h1>
    <facebook-find-us-button />
  </section>
</template>

<script>
import FacebookFindUsButton from './FacebookFindUsButton.vue'

export default {
  components: {
    FacebookFindUsButton
  },
  props: {
    pageId: String
  }
}
</script>

<style scoped>
.feed {
  margin: 50px 0;
}

.feed > h1 {
  text-align: center;
}
</style>