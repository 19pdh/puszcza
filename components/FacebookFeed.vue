<template>
  <section class="feed">
    <h1>Zobacz co siÄ™ dzieje!</h1>
    <FacebookFindUsButton />
  </section>
</template>

<script>
import FacebookFindUsButton from "./FacebookFindUsButton.vue";

export default {
  components: {
    FacebookFindUsButton
  },
  props: {
    pageId: String
  }
};
</script>

<style scoped>
.feed {
  margin: 50px 0;
}
</style>