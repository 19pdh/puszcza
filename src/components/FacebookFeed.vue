<template>
  <section>
    <h1>Ostatnie wpisy</h1>
  </section>
</template>

<script>
export default {
  mounted() {
    fetch(
      `http://fetchrss.com/rss/5d6675308a93f8773c8b45685d66750d8a93f8bb388b4567.xml`
    ).then(console.log);
  }
};
</script>