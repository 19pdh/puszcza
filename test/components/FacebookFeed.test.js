import { shallowMount } from '@vue/test-utils'
import FacebookFeed from '../../components/FacebookFeed'

describe('FacebookFeed', () => {
  // Now mount the component and you have the wrapper
  const wrapper = shallowMount(FacebookFeed)

  it('match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
