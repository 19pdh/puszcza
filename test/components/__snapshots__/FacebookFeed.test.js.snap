// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FacebookFeed match snapshot 1`] = `
<section
  class="feed"
>
  <h1>
    Zobacz co siÄ™ dzieje!
  </h1>
   
  <facebookfindusbutton-stub />
</section>
`;
